clouds/snowflake/modules/sql/processing/_VORONOIHELPER.sql

```diff
@@ -19,18 +19,23 @@ AS $$
 
     const options = {};
     options.bbox = BBOX;
-    
+
     const featuresCollection = processingLib.featureCollection(GEOJSON.map(x => processingLib.feature(JSON.parse(x))));
-    const voronoiPolygons = processingLib.voronoi(featuresCollection, options);
-    
+
+    // Truncate the point coordinates to 5 decimals, because having two very close distinct points
+    // triggers a bug a in d3-voronoi ("Cannot read properties of null (reading 'circle')")
+    // TODO: consider adding `mutate: true` to the options for performance (but user input will be altered)
```

Comment:
We are using `mutate` in `ST_CLUSTERKMEANS` as we are passing the input through an UDF I think there will be no problem if you want to add it.